js/src/components/contentAnalysis/SeoAnalysis.js

```diff
@@ -97,7 +108,8 @@ class SeoAnalysis extends React.Component {
 					subTitle={ this.props.keyword }
 				>
 					<ExplanationText>
-						{ __( "A focus keyword is the term (or phrase) you'd like to be found with in search engines. Enter it below to see how you can improve your text for this term.", "wordpress-seo" ) + " " }
+						{ __( "A focus keyword is the term (or phrase) you'd like to be found with in search engines. " +
```

Comment:
I believe there should be a comma between `with` and `in`.

Reply:
I suspect there shouldn't. @moorscode ?

Reply:
Jimmy's english suggestions are the things that I always apply.

Reply:
Now if only you had spelled english as English ;)